### PR TITLE
Fix for token issuer type issue

### PR DESCRIPTION
**Pull request by imsuneth**
Fix for https://github.com/wso2/api-manager/issues/703

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConstants.java [hunk 1]
```diff
@@ -566,6 +566,7 @@ public final class APIConstants {
     public static final String DEFAULT_ORGANIZATION_CLAIM_NAME = "http://wso2.org/claims/organization";
     public static final String DEFAULT_TOKEN_TYPE = "DEFAULT";
     public static final String TOKEN_TYPE_OAUTH = "OAUTH";
+    public static final String DEFAULT_TOKEN_TYPE_NEW = "Default";
```
Comment (chamilaadhi) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConstants.java:569]: Shall we add a meaningful name

Comment (imsuneth) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConstants.java]: Added in https://github.com/wso2/carbon-apimgt/pull/11497/commits/daa309d5b1703883b0810165a12fc117b451ce28

**Review by chamilaadhi — APPROVED** [review]

**Comment by SanojPunchihewa** [comment]
Shall we mention the public git issue instead of the apim-internal?

**Comment by imsuneth** [comment]
> Shall we mention the public git issue instead of the apim-internal?

done.

**Pull request merged**